proposals/fluentd.adoc

```diff
@@ -0,0 +1,112 @@
+== Fluentd Proposal
+
+*Name of project:* Fluentd
+
+*Description:* Fluentd is an open source data collector for unified logging layer. Fluentd's 600+ plugins connect it to many data sources and data outputs. Fluentd has a large adopter community consisting of Atlassian, LINE, Microsoft, Nintendo, GREE (and more).
+
+*Sponsor / Advisor from TOC*: Brian Grant <briangrant@google.com>
+
+*Unique Identifier*: fluentd
+
+*License*: Apache License v2.0
+
+*Source control repositories*:
+
+The donation includes all repositories on the GitHub fluent organization (https://github.com/fluent/):
```

Comment:
This is a pretty huge list - could we have a brief description of these? (grouping them might make sense, not necessarily one for each)